ozaria/site/components/common/BaseListItem.vue

```diff
@@ -5,7 +5,7 @@
 </template>
```

Comment:
Moved this component to common folder since its used by interactives as well as cinematics